packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -689,6 +689,70 @@ const deployStage = pipeline.addStage({
 });
 ```
 
+#### Invalidating the CloudFront cache when deploying to S3
+
+There is currently no native support in CodePipeline for invalidating a CloudFront cache after deployment.
+One workaround is to add another build step after the deploy step,
+and use the AWS CLI to invalidate the cache:
+
+```ts
```

Comment:
Maybe add necessary additional import of `import * as iam from '@aws-cdk/aws-iam'`